web/assets/css/globals.less

```diff
@@ -28,3 +28,18 @@ h3 {
   font-size: 14px;
 }
 
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: #3d3c3c00;
```

Comment:
this is not a real color, its transparent notice the alpha value being 00. What's a better way to use transparency?

Reply:
I think `rgba(RRR, GGG, BBB, A);` is the best format for this.

In this case, if we want the opacity to be 0 on this element, I think you'd want this code:
`background: rgba(61, 60, 60, 0);`

(Technically you could replace the RGB values with whatever as long as the property is being set to fully transparent, I'm just translating literally from the hex color code above.)

Reply:
ty